### ISSUE TITLE
Access Languages and cuisine directly

### ISSUE THREAD

**Issue by pauldardeau** [anushaunnam/First-project-git#2]
Instead of iterating through each top-level element for person, you can directly access Languages list and cuisine list.  Example: v['cuisine'].  You can also test for existence of cuisine list too.

if 'cuisine' in v:
   cuisine_list = v['cuisine']
   if 'Italian' in cuisine_list:
      print 'this person likes Italian...'

**Comment by anushaunnam** [comment]
Yes, even I thought that I am iterating through the whole dictionary unnecessarily. Will modify the code accordingly in the next version of my program.
